README.md

```diff
@@ -185,20 +185,20 @@ and/or other countries."
 Thank you to all contributors and a special thanks to:
 
 emeritus reviewers:
```

Comment:
I thought emeritus_reviewers doesn't exist in OWNERs file.

Reply:
Correct. The new tool does not parse those fields, but Richard thought that we still should thank these people, so we added them to README instead. :)

Reply:
Thanks for the context. I didn't know about that discussion - I am fine with the decision.

Reply:
Yeah, sorry about that, should have put more context into the ticket. It was all in my head only. :D

Reply:
This list is not aligning with the OWNER file's content. It'd be good to make them consistent (and then delete those from the OWNERs list in a follow-up PR).

Reply:
It does not, because they are different people. We had to rename the reviewers to approvers and remove the approvers and the maintainers. :D And put them into the Readme. Because the new tool was unable to parse either. :/

Reply:
There is `emeritus_reviewers:` section in the OWNERS file.

Reply:
Are you sure you are looking at the right revision? We removed that section in a previous PR because of the breaking owners tool.

Reply:
Yes looking at the PR https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3115/files merged.
My comment holds, we can use the list that is deleted in the PR merged.

Reply:
Ah you mean there was a reviewers section. :D Okay, so, we merged that with, errr... the maintainers and pull them together that's the discrepancy.  But honestly, I'm a bit confused now here. :D I'm hoping @richardcase can shed a bit of better light on this. :)

Reply:
I just saw the comments from the other issue, ok this is the merged list. LGTM. Thanks for the PR.